Pod/Native/NativeTestCase.swift

```diff
@@ -13,6 +13,12 @@ import XCTest
 
 public class NativeTestCase : XCTestCase {
     
+    // if you want to subclass this class without implementing any scenarios
```

Comment:
If you have shared setUp / tearDown for a collection of tests you would rather subclass this test and add common setUp than copy-paste it everywhere. I can agree that is not the most beautiful solution tho. I will try to prettify it later...